Fix errors on test import for CPU-only machines:

>    if torch.get_device_module()._initialized:
>       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> AttributeError: module 'torch.cpu' has no attribute '_initialized'. Did you mean: 'is_initialized'?

Use commit 00bc580597ad578de53960ff8e909f5410f7137d from Jean Schmidt <contato@jschmidt.me>

diff --git a/test/distributed/test_c10d_common.py b/test/distributed/test_c10d_common.py
--- a/test/distributed/test_c10d_common.py
+++ b/test/distributed/test_c10d_common.py
@@ -2890,7 +2890,7 @@ class ThreadLocalSafetyLintTest(TestCase):
 
 if __name__ == "__main__":
     if device_type != "cpu":
-        if torch.get_device_module()._initialized:
+        if getattr(torch.get_device_module(device_type), "_initialized", False):
             raise AssertionError(
                 f"test_distributed must not have initialized {device_type} context on main process"
             )
diff --git a/test/distributed/test_store.py b/test/distributed/test_store.py
--- a/test/distributed/test_store.py
+++ b/test/distributed/test_store.py
@@ -1220,7 +1220,7 @@ class TestClientProtocol(TestCase):
 
 if __name__ == "__main__":
     if device_type != "cpu":
-        if torch.get_device_module()._initialized:
+        if getattr(torch.get_device_module(device_type), "_initialized", False):
             raise AssertionError(
                 f"test_distributed must not have initialized {device_type} context on main process"
             )
